### DISCUSSION TITLE
nginx Connection reset by peer) while reading upstream

### DISCUSSION THREAD

**Discussion by marvinville** [ory/hydra#3315] [Q&A]
Hi all,

So I was trying to follow this tutorial from ORY Hydra: https://www.ory.sh/docs/hydra/self-hosted/deploy-hydra-example

From here, I was able to set up everything up correctly and what's left to do is to access this URL:

https://oauth2.example.com/health?secret=CHANGE-ME-INSECURE-PASSWORD **(in my case it's https://sso.sunwatt.io)**

However, I am receiving this error 400 on browser. Upon checking the log file from nginx, I am getting this:

**nginx Connection reset by peer) while reading upstream**

Can someone please help me?
![z2](https://user-images.githubusercontent.com/45863966/197943541-76e359ac-1596-4fb6-9f38-edc78153056a.PNG)
![z](https://user-images.githubusercontent.com/45863966/197943542-517da339-fbe8-4c73-b4fd-029063005028.PNG)
![Screenshot (1263)](https://user-images.githubusercontent.com/45863966/197943561-9fd6af68-c699-4181-bd95-d901afd72e01.png)
![Screenshot (1263)](https://user-images.githubusercontent.com/45863966/197943715-8b8e385a-506e-42cb-8f24-982ce347effe.png)

TIA

**Comment by DaveSmithh** [comment]
Hey, I am facing similar situation have you found any solution yet ?

**Comment by DaveSmithh** [comment]
@marvinville

**Reply by marvinville** [reply]
Hey @DaveSmithh 

No, still facing the issue

**Reply by vinckr** [reply]
Hey @DaveSmithh 
apologies for the late reply. 
Can you confirm all services are running correctly and all the configuration is set up like described in the guide? Is it something we need to update in the guide?
